### PR TITLE
Add new pulp repo tags

### PR DESCRIPTION
**Pull request by MoteHue**

### CODE REVIEW

**Comment by markgoddard** [comment]
Do we need some container image tags as well?

**Comment by jovial** [comment]
I've got some package versions bumps for the release notes:

```
[will@juno stackhpc-release-train-package-versions]$ jq '. | map(select(.name=="kernel")) | sort_by(.version, .release) | last' centos/8-stream/BaseOS/x86_64/os/*/packages.json
{
  "arch": "x86_64",
  "name": "kernel",
  "release": "408.el8",
  "version": "4.18.0"
}
{
  "arch": "x86_64",
  "name": "kernel",
  "release": "448.el8",
  "version": "4.18.0"
}
```

```
[will@juno stackhpc-release-train-package-versions]$ jq '. | map(select(.name | test( "^ovn22.06[^-]*$"))) | sort_by(.version, .release, .name) | last' centos/8-stream/nfv/x86_64/openvswitch-2/*/packages.json
{
  "arch": "x86_64",
  "name": "ovn22.06",
  "release": "24.el8s",
  "version": "22.06.0"
}
{
  "arch": "x86_64",
  "name": "ovn22.06",
  "release": "24.el8s",
  "version": "22.06.0"
}

```
NOTE: OVN version hardcoded to 22.06 here: https://github.com/stackhpc/stackhpc-kayobe-config/blob/stackhpc/wallaby/etc/kayobe/kolla.yml#L328

```
[will@juno stackhpc-release-train-package-versions]$ jq '. | map(select(.name | test( "^openvswitch[^-]*$"))) | sort_by(.version, .release) | last' centos/8-stream/nfv/x86_64/openvswitch-2/*/packages.json
{
  "arch": "x86_64",
  "name": "openvswitch2.17",
  "release": "31.el8s",
  "version": "2.17.0"
}
{
  "arch": "x86_64",
  "name": "openvswitch2.17",
  "release": "71.el8s",
  "version": "2.17.0"
}
```

libvirt remains the same

**Comment by jovial** [comment]
> Do we need some container image tags as well?

They are building here: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/4115798154

**Comment by markgoddard** [comment]
> > Do we need some container image tags as well?
> 
> They are building here: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/4115798154

Ok. Bear in mind bifrost is built separately, and will end up with a different tag.

**Comment by jovial** [comment]
Looks good. I will merge into `https://github.com/stackhpc/stackhpc-kayobe-config/tree/refresh/multinode-load-balancers`. Which is the wallaby/branch merged in multinode-load-balancers (wallaby multinode environment).

**Comment by jovial** [comment]
Lets add a release note saying packages have being refreshed and summarise versions of these packages:

-  kernel version
- libvirt version
- ovs version
- ovn version

**Comment by markgoddard** [comment]
> Lets add a release note saying packages have being refreshed and summarise versions of these packages:
> 
> * kernel version
> * libvirt version
> * ovs version
> * ovn version

Yes, let's :)

**Comment by MoteHue** [comment]
> > Lets add a release note saying packages have being refreshed and summarise versions of these packages:
> > 
> > * kernel version
> > * libvirt version
> > * ovs version
> > * ovn version
> 
> Yes, let's :)

I've added the latest releases and versions from [this repo](https://github.com/stackhpc/stackhpc-release-train-package-versions). I don't know why there's so many different versions of each listed, so would appreciate if this can be double checked :)

File: releasenotes/notes/release-train-02-2023-package-refresh-9de8049f8cb08a5e.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+features:
+  - |
+    The package repositories have been refreshed. Key packages to note are:
```
Comment (markgoddard) [inline releasenotes/notes/release-train-02-2023-package-refresh-9de8049f8cb08a5e.yaml]:
```suggestion
    Package repositories and container images for CentOS Stream based deployments have been updated.
    Key packages to note are:

```

Comment (markgoddard) [reply releasenotes/notes/release-train-02-2023-package-refresh-9de8049f8cb08a5e.yaml]: Note the empty line, required by RST before lists

**Review by markgoddard — CHANGES_REQUESTED** [review]

**Review by markgoddard — APPROVED** [review]

File: releasenotes/notes/release-train-02-2023-package-refresh-9de8049f8cb08a5e.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+features:
+  - |
+    Package repositories and container images for CentOS Stream based deployments have been updated.
+    Key packages to note are:
+
+    * Kernel
+      * version: 4.18.0
+      * release: 408.el8
```
Comment (priteau) [inline releasenotes/notes/release-train-02-2023-package-refresh-9de8049f8cb08a5e.yaml:9]: Is the kernel version correct? 408 was released in July and already present in the last package sync. I would have expected 448: https://koji.mbox.centos.org/koji/buildinfo?buildID=23410

File: releasenotes/notes/release-train-02-2023-package-refresh-9de8049f8cb08a5e.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    * Libvirt
+      * version: 7.6.0
```
Comment (priteau) [inline releasenotes/notes/release-train-02-2023-package-refresh-9de8049f8cb08a5e.yaml:11]: The nova-libvirt container images we built recently were all using Libvirt 8.0.0. Did you change the configuration to get Libvirt from somewhere else?

**Review by priteau — COMMENTED** [review]
This was merged already but I think there are errors in the release note.

**Pull request merged**